<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispatchOrderSign</class>
 <widget class="QDialog" name="DispatchOrderSign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DispatchOrderSign</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,3,2,0,0">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>命令类型名称:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="orderTypeLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>命令号码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="orderNumLabel">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>日期:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="orderDateLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>发令时刻:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="orderTimeLabel">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>发令单位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sendAgencyLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>调度员姓名:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sendNameLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>命令内容:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="contentEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tableFrame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>签收后打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>需他人阅读</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>签收人姓名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>72</width>
         <height>24</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refuseSignBtn">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>拒签</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signBtn">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>签收</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
